lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1689,6 +1689,8 @@ def main():
                 # has to be poweredoff first
                 pyv.set_powerstate(vm, 'poweredoff', module.params['force'])
             result = pyv.remove_vm(vm)
+        elif module.params['snapshot_op']:
```

Comment:
I simply restored what was there so that it works again as-is.
We are going to pull this out of vmware_guest and I propose we do the cleanup then and there.